bika/lims/content/analysisrequest.py

```diff
@@ -1470,7 +1470,7 @@
             description=_("The date when the request was published"),
             visible={
                 'edit': 'visible',
-                'view': 'visible',
+                'view': 'invisible',
```

Comment:
`DatePublishedViewer` field can be totally removed. Was added in https://github.com/senaite/bika.lims/pull/159  because at that moment, there was no `DatePublished` field, but a `getDatePublished` function that retrieves the date from the `publish` transition timestamp: https://github.com/senaite/bika.lims/blob/master/bika/lims/content/analysisrequest.py#L2601-L2605

During the merge in October, [the field `DatePublished` was added](https://github.com/senaite/bika.lims/pull/289/files#diff-9d57b6da936ef609e76f1d66d7886474R1451) without taking into account that `DatePublishedViewer` field was already there. On the other hand, the name of `DatePublish` field fits better and is already tagged as `read-only`. I suggest to remove `DatePublishedViewer`, but make `DatePublished` a `ComputedField`, so the value will always be stored in the schema and we will not need to apply workarounds such as the one above in `header_table`